applications/nrf5340_audio/src/audio/audio_system.c

```diff
@@ -52,7 +53,8 @@ static void audio_gateway_configure(void)
 
 #if (CONFIG_STREAM_BIDIRECTIONAL)
 	sw_codec_cfg.decoder.enabled = true;
-	sw_codec_cfg.decoder.num_ch = SW_CODEC_MONO;
+	sw_codec_cfg.decoder.num_ch = SW_CODEC_ONE_CHANNEL;
+	sw_codec_cfg.decoder.audio_mode = SW_CODEC_MONO;
```

Comment:
I'm not convinced setting it here is enough as the number of channels is still changed later in the function. Maybe it's enough to do a check at the end of the function, and set MONO/STEREO based on the number of channels?